README.md

```diff
@@ -6,7 +6,7 @@
 [![image](https://travis-ci.org/OCR-D/core.svg?branch=master)](https://travis-ci.org/OCR-D/core)
 [![image](https://circleci.com/gh/OCR-D/core.svg?style=svg)](https://circleci.com/gh/OCR-D/core)
 [![image](https://scrutinizer-ci.com/g/OCR-D/core/badges/build.png?b=master)](https://scrutinizer-ci.com/g/OCR-D/core)
-[![Docker Automated build](https://img.shields.io/docker/automated/ocrd/pyocrd.svg)](https://hub.docker.com/r/ocrd/core/tags/)
+[![Docker Automated build](https://img.shields.io/docker/automated/ocrd/core.svg)](https://hub.docker.com/r/ocrd/core/tags/)
```

Comment:
This will cause an error until I update the core image which I intended to have done weeks ago but hopefully will get to next week...